code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
+			new /obj/item/reagent_containers/glass/bottle/tuberculosiscure(src)
+			new /obj/item/reagent_containers/glass/bottle/wizarditis(src)
+			new /obj/item/clothing/head/bio_hood/virology(src)
+			new /obj/item/clothing/suit/bio_suit/virology(src)
+			new /obj/item/tank/internals/oxygen/yellow(src)
+			new /obj/item/clothing/gloves/color/latex/nitrile(src)
+		if(108)
+			new /obj/item/circuitboard/machine/protolathe(src)
+		if(109)
+			var/list/circuits = subtypesof(/obj/item/circuitboard)
+			var/chosen_circuit = pick(circuits)
+			new chosen_circuit(src)
+		if(110)
+			new /obj/item/clothing/head/bronze(src)
+			new /obj/item/clothing/shoes/bronze(src)
+			new /obj/item/clothing/suit/bronze(src)
+			new /obj/item/stack/tile/bronze/thirty(src)
+		if(111)
+			new /obj/item/clothing/accessory/medal/bronze_heart(src)
+			new /obj/item/clothing/accessory/medal/conduct(src)
+			new /obj/item/clothing/accessory/medal/plasma(src)
+			new /obj/item/clothing/accessory/medal/plasma/nobel_science(src)
+			new /obj/item/clothing/accessory/medal/ribbon/cargo(src)
+			new /obj/item/clothing/accessory/medal/silver/excellence(src)
+			new /obj/item/clothing/accessory/medal/silver/security(src)
+			new /obj/item/clothing/accessory/medal/silver/valor(src)
+		if(112)
+			new /obj/item/clothing/accessory/medal/gold/heroism(src)
+			new /obj/item/clothing/accessory/medal/gold/captain(src)
+			new /obj/item/reagent_containers/food/drinks/trophy/gold_cup(src)
+			new /obj/item/reagent_containers/food/snacks/grown/apple/gold(src)
+		if(113)//Lite, mid and heavy are Snowndin lootdrops. Putting a spawner into the chest result things spawn on top of it instead of inside.
+			switch(pickweight(list("heavystuff1" = 1, "heavystuff2" = 1, "heavystuff3" = 1, "heavystuff4" = 1, "heavystuff5" = 1, "heavystuff6" = 1, "heavystuff7" = 1, "heavystuff8" = 1)))
+				if("heavystuff1")
+					new /obj/item/twohanded/singularityhammer(src)
+				if("heavystuff2")
+					new /obj/item/twohanded/mjollnir(src)
+				if("heavystuff3")
+					new /obj/item/twohanded/fireaxe(src)
+					new /obj/item/organ/brain/alien(src)
+				if("heavystuff4")
+					new /obj/item/uplink/old(src)
+				if("heavystuff5")
+					new /obj/item/grenade/clusterbuster/spawner_manhacks(src)
+				if("heavystuff6")
+					new /obj/item/borg/upgrade/syndicate(src)
+				if("heavystuff7")
+					new /obj/item/borg/upgrade/selfrepair(src)
+				if("heavystuff8")
+					new /obj/item/book/granter/spell/charge(src)
+		if(114)
+			for(var/food in 5)
+				food = pick(subtypesof(/obj/item/reagent_containers/food/snacks))
+				new food(src)
+			new /obj/effect/spawner/lootdrop/three_course_meal(src)
+		if(115)
+			var/list/type_list = subtypesof(/obj/item)//Yes, a_gift, but you at least need to put a little effort to get the contents.
+			var/chosen_type = pick(type_list)
+			new chosen_type(src)
+		if(116)
+			switch(pickweight(list("litestuff" = 1, "litestuff2" = 1, "litestuff3" = 1, "litestuff4" = 1, "litestuff5" = 1)))
```

Comment:
No need for pickweight if all items are weight 1.

Use pick("litestuff","litestuff2"...